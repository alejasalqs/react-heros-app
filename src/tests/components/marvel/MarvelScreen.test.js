import React from 'react';
import { shallow } from "enzyme";
import { MarvelScreen } from "../../../components/marvel/MarvelScreen";

describe("Pruebas en el componente <MarvelScreen />", () => {
  test("Debe hacer match con el snapshot ", () => {
    const wrapper = shallow(<MarvelScreen />);
    expect(wrapper).toMatchSnapshot();
  });
});
