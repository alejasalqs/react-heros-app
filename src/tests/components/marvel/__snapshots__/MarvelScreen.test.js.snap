// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en el componente <MarvelScreen /> Debe hacer match con el snapshot  1`] = `
<Fragment>
  <h1>
    Marvel Heros
  </h1>
  <hr />
  <HeroList
    publisher="Marvel Comics"
  />
</Fragment>
`;
